staging/src/k8s.io/apiserver/pkg/util/flag/ciphersuites_flag.go

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package flag
+
+import (
+	"fmt"
+	"strings"
+)
+
+// CipherSuitesFlag maps strings into tls package cipher constants in
+// https://golang.org/pkg/crypto/tls/#pkg-constants
+// To parse cipher suite parameters
+type CipherSuitesFlag map[string]uint16
+
+func NewCipherSuitesFlag() *CipherSuitesFlag {
+	return &CipherSuitesFlag{
+		"TLS_RSA_WITH_RC4_128_SHA":                0x0005,
```

Comment:
see https://github.com/liggitt/origin/blob/master/pkg/cmd/server/crypto/crypto.go#L72-L112 for an example, and https://github.com/liggitt/origin/blob/master/pkg/cmd/server/crypto/crypto_test.go#L16-L54 for a test to make sure the allowed values stays up to date with the ciphers supported by the TLS stack